main.go

```diff
@@ -299,17 +299,16 @@ func loadAPIEndpoints(muxer *mux.Router) {
 	}).Info("Initialising Tyk REST API Endpoints")
 
 	// set up main API handlers
-	r.HandleFunc("/reload/group", allowMethods(groupResetHandler, "GET"))
-	r.HandleFunc("/reload/", allowMethods(resetHandler(nil), "GET"))
+	r.HandleFunc("/reload/group{_:/?}", allowMethods(groupResetHandler, "GET"))
+	r.HandleFunc("/reload{_:/?}", allowMethods(resetHandler(nil), "GET"))
 
 	if !isRPCMode() {
 		r.HandleFunc("/org/keys/{keyName:.*}", allowMethods(orgHandler, "POST", "PUT", "GET", "DELETE"))
 		r.HandleFunc("/keys/policy/{keyName:.*}", allowMethods(policyUpdateHandler, "POST"))
-		r.HandleFunc("/keys/create", allowMethods(createKeyHandler, "POST"))
-		r.HandleFunc("/apis", allowMethods(apiHandler, "GET", "POST", "PUT", "DELETE"))
-		r.HandleFunc("/apis/{apiID:.*}", allowMethods(apiHandler, "GET", "POST", "PUT", "DELETE"))
-		r.HandleFunc("/health/", allowMethods(healthCheckhandler, "GET"))
-		r.HandleFunc("/oauth/clients/create", allowMethods(createOauthClient, "POST"))
+		r.HandleFunc("/keys/create{_:/?}", allowMethods(createKeyHandler, "POST"))
+		r.HandleFunc("/apis{_:/?}{apiID:.*}", allowMethods(apiHandler, "GET", "POST", "PUT", "DELETE"))
```

Comment:
Just a thought, since "/" is optional here, does it means that "/api{apiID}" (without slash between path and id) will be valid? It is not the critical, just want to confirm.